tests_e2e/test_suites/agent_not_provisioned.yml

```diff
@@ -6,7 +6,7 @@ name: "AgentNotProvisioned"
 tests:
   - "agent_not_provisioned/agent_not_provisioned.py"
 images: "random(endorsed)"
-template: "agent_not_provisioned/template.py"
```

Comment:
renamed the generic "template.py" file to a file name matching the class defined in it (now tests define a class derived from UpdateArmTemplate)